pkg/apis/compliance/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -294,6 +295,17 @@ func (in *ComplianceScanSettings) DeepCopyInto(out *ComplianceScanSettings) {
 		*out = new(bool)
 		**out = **in
 	}
+	if in.ScanLimits != nil {
+		in, out := &in.ScanLimits, &out.ScanLimits
+		*out = new(map[v1.ResourceName]resource.Quantity)
```

Comment:
huh, wrong autogenerated code? That's a first for me..

Reply:
@mrogers950 any idea? I don't know what I'm doing wrong

Reply:
I decided to use a map in the CRD definition rather than the Resource type and now everything seems to work. From the user perspective this is the same, even for pods, the explain strings shows as a map:
```
oc explain pod.spec.containers.resources.limits
IND:     Pod                                                                  
VERSION:  v1                                                                   
                                                                               
FIELD:    limits <map[string]string>
...
```